hardware/products/ucm/g400.md

```diff
@@ -1,5 +1,250 @@
+| 7             | Reserved                      | --------------------------------------------- |
+| 8             | Ethernet RX+                  | ETH PHY RX+                                   |
+| 9             | Reserved                      | --------------------------------------------- |
+| 10            | Indicator A                   | ETH PHY LED SPEED                             |
+| 11            | Indicator B                   | ETH PHY LED LINK                              |
+| 12            | Reserved                      | --------------------------------------------- |
+| 13            | GND                           | GND                                           |
+| 14            | DCMI D0                       | --------------------------------------------- |
+| 15            | DCMI D1                       | --------------------------------------------- |
+| 16            | DCMI D2                       | --------------------------------------------- |
+| 17            | DCMI D3                       | --------------------------------------------- |
+| 18            | DCMI D4                       | --------------------------------------------- |
+| 19            | DCMI D5                       | --------------------------------------------- |
+| 20            | Analog 3.3V                   | 3.3V                                          |
+| 21            | DCMI D6                       | --------------------------------------------- |
+| 22            | DCMI D7                       | --------------------------------------------- |
+| 23            | DCMI VSYNC                    | --------------------------------------------- |
+| 24            | DCMI HSYNC                    | --------------------------------------------- |
+| 25            | DCMI PIXCLK                   | --------------------------------------------- |
+| 26            | DCMI XCLK                     | --------------------------------------------- |
+| 27            | GND                           | GND                                           |
+| 28            | PWM E                         | --------------------------------------------- |
+| 29            | PWM F                         | --------------------------------------------- |
+| 30            | PWM G                         | --------------------------------------------- |
+| 31            | PWM H                         | --------------------------------------------- |
+| 32            | Analog VREF+                  | 3.3V                                          |
+| 33            | Reserved                      | --------------------------------------------- |
+| 34            | 5V                            | --------------------------------------------- |
+| 35            | Module Specific 4             | PB3                                           |
+| 36            | Module Specific 5             | PB4 (DNP)                                     |
```

Comment:
DNP?

Reply:
I am not certain why that is there. It was in our documentation.